.travis.yml

```diff
@@ -23,6 +23,7 @@ after_success:
 before_deploy:
```

Comment:
I think you need to modify the _.travis/update_checkstyle_thresholds_ script to account for the Checkstyle thresholds in the lobby subproject.

Reply:
To be sure, we have zero violation allowance in lobby sub-project:
```
checkstyleMain {
    maxWarnings = 0
    source sourceSets.main.output.resourcesDir
}

checkstyleTest {
    maxWarnings = 0
    source sourceSets.test.output.resourcesDir
}
```

Do we still need to update the `update_checkstyle_thresholds` script in that case?

Reply:
I was under the assumption that more code from `game-core` will be migrated to this subproject in the future.  As that code will most likely have Checkstyle violations, the current solution is fine if the intention is to fix the violations during the move.  Otherwise, we'll need to support non-zero thresholds.